.bazelrc

```diff
@@ -129,6 +128,13 @@ build:auto-release --config=remote
 build:auto-release --remote_instance_name=buildbuddy-io/buildbuddy/auto-release
 build:auto-release -c opt
 
+# On multi-user machines, specify HERMETIC_CC_TOOLCHAIN_CACHE_PREFIX in your
+# repo_env in user.bazelrc to avoid permissions errors.
```

Comment:
We should add a note here to clarify that this currently only works with Linux x86_64 and not with MacOS.
It will help curious folks navigate these configs a bit easier rather than running into some of our un-supported use cases.

Reply:
Good point.